src/modules/Settings/PersonalInfo/PersonalInfoImports.ts

```diff
@@ -0,0 +1,53 @@
+import { IInputProps } from "../../../shared/Forms/Form";
+
+export const personalInfoForm: IInputProps[] = [
+	{
+	  label: "Full Name",
+	  placeholder: "Your Full Name",
+	  type: "input",
+	  value: [],
+	  isRequired: true,
+	},
+	
+	{
+	  label: "Phone Number",
+	  placeholder: "09012345678",
+	  type: "input",
+	  value: [],
+	  isRequired: true,
+	},
+	
+	{
+	  label: "Email",
+	  placeholder: "Your Email",
+	  type: "input",
+	  value: [],
+	  isRequired: true,
+	},
+	
+	{
+	  label: "Role",
+	  placeholder: "Your role",
+	  type: "input",
+	  value: [],
+	  isRequired: true,
+	},
+	
+  ];
```

Comment:
the form can be a reusable component... take away the duplicate.. there is another one at `organizationForm` or something like that

Reply:
The form is a reuseable component, I'm just abstracting the props to be passed into away from the component.

This is just me defining the inputs to be created by the `<Form/>` component, which is in my shared module, It has to be done that way because i'm using the form to handle filter inputs too

Reply:
ok...good one.
if I am correct, you make your Form component to be so much generic that you are using it for filters... it is not too cool... there is a principle in software development that says `SINGLE RESPONSIBLITY` let the form answer to single responsiblity... define a new component to handle filter...

Reply:
Done